src/test/java/me/dinowernli/grpc/prometheus/integration/MonitoringClientInterceptorIntegrationTest.java

```diff
@@ -53,126 +53,176 @@ public void tearDown() throws Throwable {
   }
 
   @Test
   public void bidiStreamRpcMetrics() throws Throwable {
     StreamObserver<HelloProto.HelloRequest> requestStream =
-        createClientStub(CHEAP_METRICS).sayHelloBidiStream(responseRecorder);
+            createClientStub(CHEAP_METRICS).sayHelloBidiStream(responseRecorder);
     requestStream.onNext(REQUEST);
     requestStream.onNext(REQUEST);
     requestStream.onCompleted();
 
     responseRecorder.awaitCompletion();
 
-    assertThat(extractMetricValue("grpc_client_started_total")).isWithin(0).of(1);
-    assertThat(extractMetricValue("grpc_client_msg_received_total")).isWithin(0).of(2);
-    assertThat(extractMetricValue("grpc_client_msg_sent_total")).isWithin(0).of(2);
+    assertThat(
+            findRecordedMetricOrThrow("grpc_client_started").samples.stream()
+                    .filter(s -> s.name.equals("grpc_client_started_total"))
+                    .findFirst().get().value)
+            .isWithin(0).of(1);
+
+    assertThat(
+            findRecordedMetricOrThrow("grpc_client_msg_received").samples.stream()
+                    .filter(s -> s.name.equals("grpc_client_msg_received_total"))
+                    .findFirst().get().value)
+            .isWithin(0).of(2);
+
+    assertThat(
+            findRecordedMetricOrThrow("grpc_client_msg_sent").samples.stream()
+                    .filter(s -> s.name.equals("grpc_client_msg_sent_total"))
+                    .findFirst().get().value)
+            .isWithin(0).of(2);
 
     Collector.MetricFamilySamples handled = findRecordedMetricOrThrow("grpc_client_completed");
-    assertThat(handled.samples).hasSize(1);
-    assertThat(handled.samples.get(0).labelValues).containsExactly(
-        "BIDI_STREAMING",
-        HelloServiceImpl.SERVICE_NAME,
-        HelloServiceImpl.BIDI_STREAM_METHOD_NAME,
-        "OK", // TODO: These are the "code" and "grpc_code" labels which are currently duplicated. "code" should be deprecated in a future release.
-        "OK");
-    assertThat(handled.samples.get(0).value).isWithin(0).of(1);
+    assertThat(handled.samples).hasSize(2);
+    Collector.MetricFamilySamples.Sample totalSample =
+            handled.samples.stream().filter(s -> s.name.equals("grpc_client_completed_total")).findFirst().get();
+    assertThat(totalSample.labelValues).containsExactly(
+            "BIDI_STREAMING",
+            HelloServiceImpl.SERVICE_NAME,
+            HelloServiceImpl.BIDI_STREAM_METHOD_NAME,
+            "OK", // TODO: These are the "code" and "grpc_code" labels which are currently duplicated. "code" should be deprecated in a future release.
+            "OK");
+    assertThat(totalSample.value).isWithin(0).of(1);
   }
 
   @Test
   public void noHistogramIfDisabled() throws Throwable {
     createClientStub(CHEAP_METRICS).sayHello(
-        HelloProto.HelloRequest.getDefaultInstance(), responseRecorder);
+            HelloProto.HelloRequest.getDefaultInstance(), responseRecorder);
```

Comment:
Can we revert all these formatting changes (here and elsewhere in the PR)?

Reply:
Done.